Gemfile

```diff
@@ -5,12 +5,13 @@ gem 'sinatra'
 gem 'thin'
 gem 'json'
 gem 'sequel'
-gem 'sqlite3'
+
 
 group :development do
   gem 'rerun'
   gem 'tux'
-  gem 'hirb'	
+  gem 'hirb'
+  gem 'sqlite3'	
```

Comment:
you need to create a group for both test and development sqlite3 is also needed for test
```
group :development, :test do
  gem 'sqlite3'
end
```